web/src/main/webapp/WEB-INF/jsp/admin/viewFullStudy.jsp

```diff
@@ -76,9 +76,9 @@
 <h1 style="margin: 0em; "><span class="title_manage"><c:out value="${studyToView.name}"/></span></h1></br>
 
 
-<fmt:message key="download_study_meta" bundle="${restext}"/>
-
-(<a href="ListStudySubjects">Subject Matrix</a>).
+<a href="javascript:openDocWindow('DownloadStudyMetadata?studyId=<c:out value="${studyToView.id}"/>');"><fmt:message key="download_study_meta" bundle="${restext}"/></a>.
+<fmt:message key="get_subject_oid_from_matrix_show_more" bundle="${restext}"/>
```

Comment:
This text is:
    For Study Subject OIDs select show more in the
would probably read better as:
    For Study Subject OIDs click "Show More" in the

Reply:
Are you kidding me !!! MERGE IT :)